spec/factories/edition_factory.rb

```diff
@@ -209,5 +209,9 @@
     trait :not_political do
       system_political { false }
     end
+
+    trait :pre_release do
```

Comment:
We probably don't need a trait for functionality that's only used once (/in one test). As a more minor point: it does seem a bit weird to say the edition is pre-release, since we're actually referring to the document type.

Reply:
Have removed the trait and moved the functionality into the test.